### PR TITLE
0.12.12 YAML support for Publish

### PR DESCRIPTION
**Pull request by leah-ferguson**
Added information for YAML keys added in 0.12.12

### CODE REVIEW

File: en/Plugins/Publish.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ With a fresh, empty site, all your notes will appear under the "New" section, me
 
 ==Privacy note: Obsidian Publish has no awareness (nor retention) of any notes you choose not to publish (nor any notes you subsequently remove from your published site).==
 
+#### YAML support for Publish
+
+As of 0.12.12, you can use YAML to flag notes to be published. (You can read more about YAML in Obsidian [[YAML front matter|here]].)
+
+If you set `publish: true`, then new files will be automatically selected the next time you publish changes.
+
+If you set `publish: false`, then the file will not be visble when publishing changes, and will not show up in new/changed files. If this file had been previously published, you will see an option to delete it isntead.
```
Comment (kometenstaub) [inline en/Plugins/Publish.md:31]: There's a typo in the last word.

**Review by kometenstaub — CHANGES_REQUESTED** [review]

File: en/Plugins/Publish.md (hunk 1 above)
Comment (leah-ferguson) [reply en/Plugins/Publish.md:31]: Thanks!

**Review by kometenstaub — APPROVED** [review]
:+1:

**Comment by ericaxu** [comment]
Thank you for your amazing work!

**Pull request merged**